src/cli/utils/users.go

```diff
@@ -17,23 +18,28 @@ func UsersMap(
 	ctx context.Context,
 	acct account.Account,
 	co control.Options,
+	counter *count.Bus,
 	errs *fault.Bus,
 ) (idname.Cacher, error) {
-	au, err := makeUserAPI(acct, co)
+	au, err := makeUserAPI(acct, counter, co)
 	if err != nil {
 		return nil, clues.Wrap(err, "constructing a graph client")
 	}
 
 	return au.GetAllIDsAndNames(ctx, errs)
 }
 
-func makeUserAPI(acct account.Account, co control.Options) (api.Users, error) {
+func makeUserAPI(
+	acct account.Account,
+	counter *count.Bus,
+	co control.Options,
```

Comment:
nit: this feels a little weird since it's got the opposite ordering for `counter` and `co` than `m365.NewController`

Reply:
aye, will fix

Reply:
All ordering should now follow:
1. regular params
2. busses
  a. counter
  b. fault 
3. variadics